### PR TITLE
Add RubyGems Update for December 2021

### PR DESCRIPTION
**Pull request by lauragift21**
Hi, @deivid-rodriguez  and @sonalkr132 can you let me know what other updates I'm missing from last month's work. 

So far I only have one from each project.

Thanks.

### CODE REVIEW

File: _posts/2022-01-14-december-rubygems-updates.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+We also worked on Bundler version switching, fixed a couple small regressions, and improved some resolution errors.
```
Comment (deivid-rodriguez) [inline _posts/2022-01-14-december-rubygems-updates.md:16]: For my side, I would change this to something like

> With this batch of releases, we finally shipped the Bundler version switching feature, which had been planned for some time now, and provided final versions to be included with new Ruby 3.1 release.

**Review by deivid-rodriguez — COMMENTED** [review]
I think one small sentence should be enough to capture last month's work. I left my suggestion inline 👍

File: _posts/2022-01-14-december-rubygems-updates.md (hunk 1 above)
Comment (lauragift21) [reply _posts/2022-01-14-december-rubygems-updates.md:16]: thanks for the input. I'll update the post.

Comment (deivid-rodriguez) [reply _posts/2022-01-14-december-rubygems-updates.md:16]: Thank you as always @lauragift21!

**Pull request merged**